### PR TITLE
prevent mix keys in filters (#936)

### PR DESCRIPTION
**Pull request by ewgRa**
According to discussion in #936 - this PR does not allow to mix Filters keys

### CODE REVIEW

**Comment by ewgRa** [comment]
Also check added that name can be only string

File: lib/Elastica/Aggregation/Filters.php [hunk 1]
```diff
@@ -10,6 +11,14 @@
  */
 class Filters extends AbstractAggregation
 {
+    const NAMED_TYPE = 1;
+    const ANONYMOUS_TYPE = 2;
+
+    /**
+     * @var int Type of bucket keys - named, or anonymous
+     */
+    private $type = null;
```
Comment (ruflin) [inline lib/Elastica/Aggregation/Filters.php]: Please use $_type for private variables.

**Comment by ruflin** [comment]
@ewgRa Looks good. Can you adjust the variable name?

**Comment by ewgRa** [comment]
@ruflin done

**Comment by ruflin** [comment]
Merged, thx.

**Pull request merged**